src/app/components/team-selector/team-selector.component.ts

```diff
@@ -0,0 +1,33 @@
+import { Component, OnInit, Input } from '@angular/core';
+import { MatchService } from '@services/match.service';
+import { Team } from '@models/team';
+import { Match } from '@models/match';
+
+@Component({
+  selector: 'app-team-selector',
+  templateUrl: './team-selector.component.html',
+  styleUrls: ['./team-selector.component.scss']
+})
+export class TeamSelectorComponent implements OnInit {
+
+  @Input() teams: Team[]
+  @Input() match: Match
+  @Input() selectedTeam: number
+
+  constructor(
+    private matchService: MatchService,
+  ) { }
+
+  ngOnInit() {
+  }
+
+  setTeam(match: Match, team: number) {
```

Comment:
I think I'd probably call this `setReferee`, even though it happens to be a team.

Reply:
On second thought, it's strange that this logic lies in this component.

Reply:
Look into changing this component to utilize the `@Output` decorator (https://angular.io/guide/inputs-outputs).

This component should not worry about what we do with the selection, but leave that to it's parent to decide. We simply want to emit the event.

Reply:
Yes, that makes much more sense! I will try to fix that :)

Reply:
@nilssonemil better? :)

Reply:
Fabulous!